metadata-ingestion/tests/integration/unity/unity_catalog_mces_golden.json

```diff
@@ -3345,16 +4034,992 @@
+        "lastRunId": "no-run-id-provided"
+    }
+},
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:databricks,system.default.quickstart_table,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "schemaMetadata",
+    "aspect": {
+        "json": {
+            "schemaName": "system.default.quickstart_table",
+            "platform": "urn:li:dataPlatform:databricks",
+            "version": 0,
+            "created": {
+                "time": 0,
+                "actor": "urn:li:corpuser:unknown"
+            },
+            "lastModified": {
+                "time": 0,
+                "actor": "urn:li:corpuser:unknown"
+            },
+            "hash": "",
+            "platformSchema": {
+                "com.linkedin.schema.MySqlDDL": {
+                    "tableSchema": ""
+                }
+            },
+            "fields": [
+                {
+                    "fieldPath": "columnA",
+                    "nullable": true,
+                    "type": {
+                        "type": {
+                            "com.linkedin.schema.NumberType": {}
+                        }
+                    },
+                    "nativeDataType": "int",
+                    "recursive": false,
+                    "isPartOfKey": false
+                },
+                {
+                    "fieldPath": "columnB",
+                    "nullable": true,
+                    "type": {
+                        "type": {
+                            "com.linkedin.schema.StringType": {}
+                        }
+                    },
+                    "nativeDataType": "string",
+                    "recursive": false,
+                    "isPartOfKey": false
+                }
+            ]
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1638860400000,
+        "runId": "unity-catalog-test",
+        "lastRunId": "no-run-id-provided"
+    }
+},
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:databricks,system.default.quickstart_table,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "browsePathsV2",
+    "aspect": {
+        "json": {
+            "path": [
+                {
+                    "id": "urn:li:container:94af9873ac56d46a4bcc995836dc15d4",
+                    "urn": "urn:li:container:94af9873ac56d46a4bcc995836dc15d4"
+                },
+                {
+                    "id": "urn:li:container:b84ba0cc36e3b2636942fc34ec5724d6",
+                    "urn": "urn:li:container:b84ba0cc36e3b2636942fc34ec5724d6"
+                }
+            ]
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1638860400000,
+        "runId": "unity-catalog-test",
+        "lastRunId": "no-run-id-provided"
+    }
+},
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:databricks,system.default.quickstart_table_external,PROD)",
```

Comment:
@mayurinehate can you please explain this bit more?

Reply:
If you look carefully, quickstart_table_external that was added as a table in unit test gets repeated with multiple different <catalog>.<schema> combinations.e.g. system.default.quickstart_table_external, system.quickstart_schema.quickstart_table_external, quickstart_catalog.quickstart_schema.quickstart_table_external. Ideally, its enough to get this table added in just one of the schemas. The mocking logic needs to be updated for this.

Reply:
we had multiple `schemas` and `catalogs`  in the test file, against which we are registering the mocks; because of that multiple entires are being generated.

For now have kept single `schema` and `catalog` and removed rest of the entires.